editor/modes/visual-editor/block.js

```diff
@@ -64,14 +65,17 @@ function VisualEditorBlock( props ) {
 			className={ className }
 		>
 			{ ( isSelected || isHovered ) && <BlockMover uid={ block.uid } /> }
-			{ isSelected && settings.controls ? (
-				<Toolbar
-					controls={ settings.controls.map( ( control ) => ( {
-						...control,
-						onClick: () => control.onClick( block.attributes, setAttributes ),
-						isActive: () => control.isActive( block.attributes )
-					} ) ) } />
-			) : null }
+			<div className="editor-visual-editor__block_controls">
```

Comment:
Should be `editor-visual-editor__block-controls` no? (`block-controls` instead of `block_controls`)